test/cli/compiler/test-migrations.js

```diff
@@ -1,35 +1,51 @@
 const test = require("tape"); // https://github.com/substack/tape
+const colorize = require('tap-colorize');
 const path = require("path");
-const process = require("process");
 const testUtils = require("../../../bin/tools/utils");
 const fsp = require("fs").promises;
 const digest = require("util").promisify(require("dirsum").digest);
-//const qx = path.resolve(__dirname, "..", "..", "..", "bin", "source", "qx");
-const qx = testUtils.getCompiler();
-const debug = false;
+//const qxCmdPath = path.resolve(__dirname, "..", "..", "..", "bin", "source", "qx");
```

Comment:
Remove line?

Reply:
I can do that. However, we should somehow make it possible to run the source version too. but you are right, this isn't the way to do it.

Reply:
Done.